internal/deviations/deviations.go

```diff
@@ -67,6 +67,9 @@ import "flag"
 
 // Vendor deviation flags.
 var (
+	NtpAssociationType = flag.Bool("ntp_association_type_enabled", false,
```

Comment:
Please be advised that NTP must be capitalized according to Go initialism.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Deviation flags must begin with `--deviations_`.

Also, the help string should explain what the OpenConfig behavior is for a fully compliant device.

Fixing these issues in #762